sdk/healthinsights/Azure.Health.Insights.RadiologyInsights/src/Generated/ContactPointSystem.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Health.Insights.RadiologyInsights
+{
+    /// <summary>
+    /// Contact Point System
+    /// see https://www.hl7.org/fhir/R4/valueset-contact-point-system.html
+    /// </summary>
+    public readonly partial struct ContactPointSystem : IEquatable<ContactPointSystem>
```

Comment:
all these new types come with the new discriminated sub types: https://github.com/Azure/azure-rest-api-specs/blob/1c63635d66ae38cff18045ab416a6572d3e15f6e/specification/ai/HealthInsights/HealthInsights.Common/model.common.fhir.resources.tsp#L318
Everything here is expected.